--- a/config.mk
+++ b/config.mk
@@ -14,9 +14,12 @@ X11LIB = /usr/lib
 XINERAMALIBS = -L${X11LIB} -lXinerama
 XINERAMAFLAGS = -DXINERAMA
 
+XFTINC = $(shell pkg-config --cflags xft)
+XFTLIB = $(shell pkg-config --libs xft)
+
 # includes and libs
-INCS = -I.  -I${X11INC}
-LIBS =  -lc -L${X11LIB} -lX11 ${XINERAMALIBS}
+INCS = -I.  -I${X11INC} ${FREETYPE2INC} ${XFTINC}
+LIBS =  -lc -L${X11LIB} -lX11 ${XINERAMALIBS} ${XFTLIB}
 
 # flags
 CPPFLAGS = -DVERSION=\"${VERSION}\" ${XINERAMAFLAGS}
